packages/puppeteer-core/src/cdp/Accessibility.ts

```diff
@@ -441,6 +448,14 @@ class AXNode {
 
     const node: SerializedAXNode = {
       role: this.#role,
+      elementHandle: async (): Promise<ElementHandle | null> => {
+        if (!this.payload.backendDOMNodeId) {
+          return null;
+        }
+        return (await this.#realm.adoptBackendNode(
+          this.payload.backendDOMNodeId
+        )) as ElementHandle<Element>;
+      },
```

Comment:
How about something like this, so we change the struct less?  
And you are can just check `'elementHandle' in X`
```suggestion
      ...{ this.payload.backendDOMNodeId ? {
        elementHandle: async (): Promise<ElementHandle> => {
          return (await this.#realm.adoptBackendNode(
            this.payload.backendDOMNodeId
          )) as ElementHandle<Element>;}
        }
        : {}
      },   
```

Reply:
I think it's better if the method is always available. I don't think we are constrained by changing the class and perhaps we could turn it into a proper class later.